modules/olmv1-adding-plain-to-fbc.adoc

```diff
@@ -86,13 +86,13 @@ The `opm render` command does not support adding plain bundles to catalogs. You
     "schema": "olm.bundle",
     "name": "example-extension.v0.0.1",
     "package": "example-extension",
-    "image": "quay.io/rashmigottipati/example-extension-bundle:v0.0.1",
+    "image": "quay.io/example-org/example-extension-bundle:v0.0.1",
     "properties": [
         {
             "type": "olm.package",
             "value": {
             "packageName": "example-extension",
-            "version": "v0.0.1"
+            "version": "0.0.1"
```

Comment:
Had to dropt he `v` here because it was failing validation:

```
$ opm validate plainbundlecatalog  

FATA[0000] error parsing bundle "example-extension.v0.0.1" version "v0.0.1": Invalid character(s) found in major number "v0" 
```

Seems related to https://github.com/operator-framework/operator-registry/issues/932. Running:

```
$ opm version

Version: version.Version{OpmVersion:"v1.30.0-2-gb1374806", GitCommit:"b1374806", BuildDate:"2023-10-24T14:02:39Z", GoOs:"darwin", GoArch:"arm64"}
```